Makefile

```diff
@@ -1,5 +1,9 @@
 GOPATH := $(shell go env GOPATH)
 GOBIN := $(GOPATH)/bin
+COMMIT := $(shell scripts/git/commit.sh)
+BUILD_LDFLAGS := -X github.com/interuss/dss/pkg/build.time=$(shell date -u '+%Y-%m-%d.%H:%M:%S') -X github.com/interuss/dss/pkg/build.commit=$(COMMIT) -X github.com/interuss/dss/pkg/build.host=$(shell hostname)
+VERSION_LDFLAGS := -X github.com/interuss/dss/pkg/version.tag=$(shell git describe --tags --abbrev=0) -X github.com/interuss/dss/pkg/version.commit=$(COMMIT)
```

Comment:
For sure, will add documentation. In summary, though, the versioning considers git tags as the single source of truth. With that, we are currently in limbo as we haven't tagged a release version. `schema_v0.0.1` was rather introduced for book-keeping purposes as far as I recall.

Reply:
Basically, I'm thinking that we'll want to use tags that aren't all of the form "vX.Y.Z" -- in this case, we tagged information about the DB schema (prior to adding migrations) and that tag is on example of a tag that doesn't follow the "vX.Y.Z" format.  Will this logic be able to selectively choose the appropriate "vX.Y.Z" tag among other tags that may not share that format?

Reply:
That's fine. We can limit the match to 'v*', thus selecting the most recent release tag.

Reply:
If we're prefix-matching, could we maybe do at least `ver1.0.0` to deconflict with other possible tags starting with `v`?  Or, it might be nicer to explicit: `codebaseversion1.0.0` despite longer tags

Reply:
I added an additional `grep` to validate the tag format.